### PR TITLE
feat(unlock-app, networks)Add link to OpenSea profile

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description

Screenshot 2023-03-30 at 16 59 06

Screenshot 2023-03-30 at 16 59 12

# Issues

Fixes #
Refs #

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Review by searchableguy — APPROVED** [review]

**Comment by julien51** [comment]
Hum, I don't think it makes sense to put the link there... Because that menu is for things that are specific to the key itself.
I think a much better place for this would be at the top of the page, below the "keychain" menu.
Screenshot 2023-03-30 at 12 18 57 PM

**Comment by julien51** [comment]
Maybe adding a menu like this one on the right with the OpenSea logo to link to OpenSea and maybe even to Etherscan for that address?
Screenshot 2023-03-30 at 12 19 38 PM

**Comment by kalidiagne** [comment]
> Hum, I don't think it makes sense to put the link there... Because that menu is for things that are specific to the key itself. I think a much better place for this would be at the top of the page, below the "keychain" menu. Screenshot 2023-03-30 at 12 18 57 PM

Not sure, the thing is for which network? Also for the `etherscan` link, if we put it outside the card, we need to have something generic, and the links are different for every network (eg. `https://goerli.etherscan.io/` vs `https://mumbai.polygonscan.com/`)

**Comment by kalidiagne** [comment]
> > Not sure, the thing is for which network?
> 
> Good point. since the link is the same for _all_ networks but testnets let's just use that?
> 
> As for Etherscan, let's use `https://blockscan.com/address/...` like we already do on the members page

**OpenSea**

Screenshot 2023-04-03 at 13 09 23

**Blockscout**

Screenshot 2023-04-03 at 13 09 29

**Comment by julien51** [comment]
Very cool! Thanks!

**Review by julien51 — APPROVED** [review]

File: unlock-app/src/components/content/KeychainContent.tsx [hunk 1]
```diff
@@ -2,13 +2,44 @@ import React from 'react'
...
+                rel="noreferrer"
+                className="hover:text-brand-ui-primary"
+              >
+                <WorldIcon size={25} />
+              </a>
+            </Tooltip>
+
+            <Tooltip tip="View Opensea Profile" label="View Opensea Profile">
+              <a
+                href={`https://opensea.io/${account}`}
```
Comment (julien51) [inline unlock-app/src/components/content/KeychainContent.tsx:32]: Actually can use please add this as a te,plate in the networks file?

File: unlock-app/src/components/content/KeychainContent.tsx [hunk 2]
```diff
@@ -2,13 +2,44 @@ import React from 'react'
...
   return (
     <AppLayout
-      title="Member Keychain"
+      title={
+        <div className="flex justify-between">
+          <h1 className="text-4xl font-bold">Member Keychain</h1>
+          <div className="flex gap-3">
+            <Tooltip tip="" label="">
+              <a
+                href={`https://blockscan.com/address/${account}`}
```
Comment (julien51) [inline unlock-app/src/components/content/KeychainContent.tsx:21]: Please add this to the networks configs?

**Review by julien51 — APPROVED** [review]

**Pull request merged**